src/routes/+page.svelte

```diff
@@ -70,12 +73,7 @@
 		}
 	}
 
-	$: if ($galleryStore.selectedTag) {
```

Comment:
@alvinosh 
This part of the code works for search with tags. 
I think it isn't related to folder selection on Linux issues.

Reply:
I updated this part myself.